src/main/java/org/jfrog/hudson/util/publisher/PublisherContext.java

```diff
@@ -57,6 +58,7 @@
     private String customBuildName;
     private boolean overrideBuildName;
     private int connectionRetry;
+    private GradlePublications gradlePublications;
```

Comment:
GradlePublications is a pipeline type, so it should not be used here. We can use a simple List<String> (or even String[]) instead.
The same goes for its usage in the ExtractorUtils class. In ExtractorUtils, it looks like we made this mistake with other pipeline types as well. This violates the modules separation principle.